.travis.yml

```diff
@@ -27,6 +27,11 @@ jobs:
     - os: osx
       node_js: node
       env: [TEST=1, TEST_ELECTRON=1, BUILD_CMD=prebuild, BUILD_GROUP=darwin-x64]
```

Comment:
@vweevers Side note. Why isn't `osx` using `if: tag is present`?

Reply:
Because that job runs tests too

Reply:
My understanding of this is that `if: tag is present` says "only invoke this if the current commit is tagged, so is probably a release that needs a prebuild". Otherwise it's just a normal test run. So osx gets test runs _and_ prebuilds, but the prebuild-only jobs (arm and alpine) only get invoked when you need a prebuild and they don't get tests.

Maybe it's a good idea to also invoke alpine and centos7 builds on normal commits, not just tagged ones, and run the tests as well inside those containers?

Reply:
Running tests in docker containers would make build times too long IMO.